atd-vzv/src/views/NotFound/UnderMaintenance.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Footer from "../nav/Footer";
+
+import { Container, Row, Navbar, Alert } from "reactstrap";
+import { responsive } from "../../constants/responsive";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+
+
+// UnderMaintance is a fallback view that is rendered when the REACT_APP_UNDER_MAINTENANCE environment variable is "true"
+// This variable can be set via the Netlify dashboard under Site Configuration in the section titled Environment variables
+// It can also be toggled in the start script defined in package.json
+// isMeasuresPath is true when the route is '/measures', the view that is serves the widgets on the austintexas.gov drupal site
+// that view should not show the header nor footer, since its an embed in another page
+const UnderMaintenance = ({ isMeasuresPath }) => {
+  return (
+    <div style={{ width: "100%" }}>
+      {!isMeasuresPath && (
+        <header role="banner">
+          <Navbar
+            light
+            className="navbar shadow-sm static-top header-navbar px-0"
+            expand="md"
+            style={{ height: `${responsive.headerHeight}px` }}
+          >
+            <Container fluid className={"px-0"}>
+              <div className="vz-logo-wrapper">
+                <h1 className="sr-only">
```

Comment:
this is a good reminder of the accessibility improvements we made from react-axe recommendations!